build/factory/memory-server-factory.js

```diff
@@ -0,0 +1,108 @@
+"use strict";
+var __awaiter = (this && this.__awaiter) || function (thisArg, _arguments, P, generator) {
```

Comment:
I know the linters don't like the use of `var`, you might want to change to `const` or to `let` instead.